apps/newsletters-ui/src/app/components/SimpleForm.tsx

```diff
@@ -150,7 +150,7 @@ export function SimpleForm<T extends z.ZodRawShape>({
 			<Box marginBottom={2}>
 				<Button
 					variant="contained"
-					onClick={handleSubmit}
+					onClick={void handleSubmit}
 					disabled={isDisabled}
```

Comment:
forgot to add - you might need to change the prop type for `handleSubmit` function to return `Promise<void>` rather than `void | `Promise<void>` - its only used in the one place (edit form) so won't affect anything else.